tracker-agent/src/main/java/c8y/trackeragent/subscription/SubscriptionApi.java

```diff
@@ -34,6 +35,6 @@ void subscribe(String tenant, ApplicationReferenceRepresentation application) {
     }
 
     String getApplicationKey() {
-        return platform.getApplicationKey();
+        return "tracker-agent-key"; // cannot be taken from Platform cast to PlatformImpl, because some integration tests would fail
```

Comment:
I see there is an application.key property in src/main/resources/application.properties, can you take the value from there?

Reply:
It should be taken from the Platform, but as I described in the comment, it cannot be. It is easier that way.

Reply:
Can't you use sth. like 
@Value("${application.key}")
private String applicationKey;

Reply:
Good idea, changed.